addons/calendar/tests/test_attendees.py

```diff
@@ -40,7 +41,7 @@ def test_attendee_added_create(self):
 
     def test_attendee_added_create_with_specific_states(self):
         """
-        When an event is created from an external calendar account (such as Google) which is not linked to an 
+        When an event is created from an external calendar account (such as Google) which is not linked to an
```

Comment:
We avoid linting in fixes to keep shorter diffs and avoid adding steps to git blaming, otherwise lgtm thanks for the fix and for the wait!